atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -62,6 +64,10 @@ def get_config_map(self):
 
         return annotations
 
+    def get_filesystem_koji_task_id(self):
+        res = self.get_result(self.workflow.prebuild_results.get(PLUGIN_ADD_FILESYSTEM_KEY, ''))
```

Comment:
Don't think you need the default empty string here. Omit second parameter to `get` letting it default to `None`.